src/utils/testHelpers.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code, as requested in the previous code review to ensure that your code is robust code and clean.